db/seeds.rb

```diff
@@ -41,6 +42,7 @@
 )
 premium.skip_confirmation!
 premium.save!
+premium.update(role: "premium")
```

Comment:
Lines 35 and 45 shouldn't be necessary after changing `User#default_to_standard` as mentioned in another Pull Request's comments.